cli-plugins/socket/socket_openbsd.go

```diff
@@ -0,0 +1,19 @@
+//go:build openbsd
+
+package socket
+
+import (
+	"net"
+	"syscall"
+)
+
+func listen(socketname string) (*net.UnixListener, error) {
+	return net.ListenUnix("unix", &net.UnixAddr{
+		Name: "@" + socketname,
```

Comment:
Curious; was there a reason to differentiate from the Darwin implementation here? Looks like the difference here with the Darwin one is that that one uses the system's temp-dir as an extra precaution from these ending up in the working-directory (which could also be the build-context);

```go
Name: filepath.Join(os.TempDir(), socketname),
```

Reply:
@thaJeztah good point. Mainly becuase it has naming darwin and nondarwin :)

But I like this way more, let me try it.

Reply:
It works like a charm

Reply:
Ah, that's great!

I'm not at my computer right now; are the openBSD and Darwin implementations now exactly the same?

If they are, perhaps we should use the same file for both then. I guess the file suffix should be renamed to something else the  (_nolinux? _other?) good suggestions for that welcome 😅

cc @laurazard

Reply:
And better to rename nodarwin as well.

But I really have not idea regarding naming.